### DISCUSSION TITLE
Action Fields - Unused space & adjustment bug

### DISCUSSION THREAD

**Discussion by RealityGamin** [laravel/nova-issues#4528] [Ideas & Feature Requests] [answered]
- Nova Version: 4.11

### Description:

Light bug, but I feel like this doesn't look right.

![image](https://user-images.githubusercontent.com/100086757/177081777-3ed77076-bcbb-46db-b276-9990d27e2587.png)

Also I think the input area should be longer, its not utilizing the full area of the modal

### Detailed steps to reproduce the issue on a fresh Nova installation:

Create a simple action with:

```
/**
     * Perform the action on the given models.
     *
     * @param  \Laravel\Nova\Fields\ActionFields  $fields
     * @param  \Illuminate\Support\Collection  $models
     * @return mixed
     */
    public function handle(ActionFields $fields, Collection $models)
    {
        return '';
    }

    /**
     * Get the fields available on the action.
     *
     * @param  \Laravel\Nova\Http\Requests\NovaRequest  $request
     * @return array
     */
    public function fields(NovaRequest $request)
    {
        return [
            Text::make('Model')->rules('required'),
            Select::make('Category')->options([
                '0' => 'Test1',
                '1' => 'Test2',
                '2' => 'Test3',
                '3' => 'Test4'
            ])->displayUsingLabels()->rules('required')
        ];
    }
```

**Comment by RealityGamin** [comment]
@crynobone Im not sure why you made it a discussion, its really a bug, take a look here

![image](https://user-images.githubusercontent.com/100086757/177114648-67e563cd-3220-41f8-8cfb-f35059710aac.png)

`md:w-1/5` - for input field name
`md:w-3/5` - for input field

missing 1/5th of the box as unused space :)

**Reply by crynobone** [reply]
When you put it as a question and suggestion, it doesn't seem like a bug report. It not in our best judgement to assume things without knowing exactly what's is wrong.

 We have dedicated template for reporting a bug

**Reply by RealityGamin** [reply]
My apologies, English is not my first language, sometimes its hard to explain properly

**Reply by davidhemphill** [reply]
I would also note that simply because we're not using all of the space of our 12-column grid, does not mean we've incorrectly implemented it. This is an intentional design decision of Nova. 🙂

**Reply by salmanhijazi** [reply]
Is there a way to override this? As it makes the form look ugly and often difficult to use.

**Reply by juse-less** [reply]
@salmanhijazi Not directly related to the action field(s) themselves, but there's a feature/update pending for size-configurable modals.
So if your problem is in modals, that could help a bit.
But it would be applied to all fields, since you'd be widening the modal.

https://github.com/laravel/nova-issues/discussions/3950#discussioncomment-3597896

**Comment by baszach** [comment]
Hi,
my some of my action modals look very squeezed too since Nova 4 e.g. with the `KeyValue` field
![image](https://user-images.githubusercontent.com/47082864/181046968-7f088a03-33d3-43f7-9324-9813fdf7d8a3.png)

but also when some field names are long
![image](https://user-images.githubusercontent.com/47082864/181047190-aaf9250b-f9b6-42b3-8000-955b3604b182.png)

**Comment by davidhemphill** [comment, marked as answer]
We added support for modal sizes in v4.14.0 (expanded in v4.15.0) using the `size` helper.
